java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -84,6 +84,20 @@ cudf::data_type timestamp_to_duration(cudf::data_type dt) {
   return cudf::data_type(duration_type_id);
 }
 
+bool decimal_type(cudf::type_id n_type) {
+  return n_type == cudf::type_id::DECIMAL32 || n_type == cudf::type_id::DECIMAL64 ;
+}
+
+cudf::data_type update_if_decimal_type(jint out_dtype, jint scale) {
```

Comment:
I don't think this name is clear.  I would rather see it be something like `make_data_type`.

Reply:
Thanks for the suggestion. done